src/RepoM.Api/Resources/RepoM.Filtering.yaml

```diff
@@ -1,3 +1,12 @@
+Private:
```

Comment:
Not sure what happened? Did you switch both items?

Reply:
Yes, but also querry is now just tag:private . This is because I've added "All" as well as which has both Work and Private.